lib/src/my_target_flutter.dart

```diff
@@ -14,21 +16,23 @@ class MyTargetFlutter {
   static const _methodCreateInterstitialAd = 'createInterstitialAd';
   static const _methodLoad = 'load';
   static const _methodShow = 'show';
-// TODO: docs
+
+  /// Initialising MyTarget ads. If[useDebugMode] = true
+  /// and [testDevices] != null will show test ads.
   Future<void> initialize({bool? useDebugMode, String? testDevices}) async {
     await _channel.invokeMethod(_methodInitialize,
         _getInitialData(useDebugMode ?? isDebug, testDevices));
   }
 
-  // TODO: docs
+  /// Creating an Interstitial ads for [slotId]
```

Comment:
```suggestion
  /// Create an Interstitial ads with [slotId]
```

ну и тоже ссылка на доку не помешает